client/src/api/v2/powerModel.js

```diff
@@ -90,3 +90,10 @@ export function setCalibration(distance) {
 export function resetCalibration() {
   emit('reset-calibration');
 }
+
+/**
+ * Send Message
+ */
+export function sendMessage(message) {
```

Comment:
I think this belongs in `api/v2/camera.js`

Reply:
Yes, my bad.